tests/src/test/scala/cats/tests/OneAndTests.scala

```diff
@@ -64,6 +64,12 @@ class OneAndTests extends CatsSuite {
   checkAll("NonEmptyList[Int]", ComonadTests[NonEmptyList].comonad[Int, Int, Int])
   checkAll("Comonad[NonEmptyList[A]]", SerializableTests.serializable(Comonad[NonEmptyList]))
 
+  test("Foldable[OneAnd[Vector, ?]]#size consistent with Vector#size") {
+    forAll { (oa: OneAnd[Vector, Int]) =>
+      oa.size should === (1L + oa.tail.size.toLong)
```

Comment:
This seems to be duplicating the implementation within the test. Instead, it might be better to do something like `oa.size should === (oa.unwrap.size)` or `oa.size should === (oa.toList.size)`.